### PR TITLE
excludes fee paid from amount in transactions list

### PR DESCRIPTION
**Pull request by hughy**
## Summary

uses transaction type to determine whether to do fee correction on IRON amount in transaction.

renames 'Fee ($IRON)' column to 'Fee Paid ($IRON)' and only displays the fee if the account paid the transaction fee.

renames 'Amount ($IRON)' column to 'Net Amount ($IRON)' and only displays non-zero values.

defines type for TransactionRow and defines separate function for constructing the row from the transaction response.

## Testing Plan

Before:
image

After:
image

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by mat-if** [comment]
Comparing master with this PR, it seems like the sends lose the net amount entirely and not just the fee
image

**Comment by hughy** [comment]
> Comparing master with this PR, it seems like the sends lose the net amount entirely and not just the fee (to confirm: all but 1 of those sends were in $IRON)

Fixed! It wasn't displaying negative net $IRON amounts, but now it's changed to show any non-zero amount:
image

File: ironfish-cli/src/commands/wallet/transactions.ts [hunk 1]
```diff
@@ -111,4 +114,42 @@ export class TransactionsCommand extends IronfishCommand {
...
+
+    let amount = BigInt(nativeAssetBalanceDelta?.delta ?? '0')
+
+    let feePaid = BigInt(transaction.fee)
+
+    if (transaction.type !== TransactionType.SEND) {
+      feePaid = 0n
+    } else {
+      amount += feePaid
+    }
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/wallet/transactions.ts:133]: Can we also adjust the amount for miners transactions too?

Comment (hughy) [reply ironfish-cli/src/commands/wallet/transactions.ts:133]: This is what I get for miner's fees testing locally: 
image

Should it be adjusted differently?

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**